src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onHandleModal: (todo: Todo) => void;
+  selectedTodo: Todo | undefined,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onHandleModal = () => {},
```

Comment:
`onHandleModal` is a required field so you don't need to assign a default value.

And think about naming too, `handler` is only for handlers, and `on` is for props.
`onOpenModal`